common/utils.js

```diff
@@ -3,7 +3,7 @@
 module.exports = {
 
   // e.g., 'Gin and Tonic' => 'gin-and-tonic'
-  formatForUrl: function (str) {
+  formatForUrl (str) {
```

Comment:
This is the ES6 object property shorthand, where `formatForUrl: function (str) {...` (ES5) turns into `formatForUrl: (str) {...` (ES6 arrow function) turns into just `formatForUrl (str) {...` (object shorthand version)